project/frontend/src/components/Authentication.js

```diff
@@ -37,9 +41,43 @@ function Authentication(props) {
     });
   }, []);
 
+  const signInWrapper = async () => {
+    let user;
+    try {
+      user = await GoogleAuth.signIn();
+    } catch (e) {
+      return;
```

Comment:
Should we throw some sort of error here?

Reply:
The `GoogleAuth.signIn()` function has some custom errors that it throws that aren't really indicative of a serious problem. For instance if the user closes out of the log in popup or denies access to the requested information, errors are thrown, so I figured it best to just ignore if an error is thrown.

Also throwing an error in this function would cause problems with all the unprotected `signIn()` calls we make throughout the different components. And even if we surrounded all those with try-catch, I don't think there would be anything to do really if we were to catch an error. The best we can hope for is just that the user tries to log in again.

Reply:
Do we even need a try catch then? Would be nice to have some sort of component that would indicate to the user that the signIn failed and to try again, but that can be a follow up :) Maybe for now we can just add a TODO to handle that case.

Reply:
Yeah I like the idea of a component to notify the user that it failed. I will add a TODO for now. For now the try-catch does prevent errors from being logged if the user closes the login popup without signing in.

Reply:
Sg! Let's maybe also add a comment in the catch block saying why we don't handle the error here